ansible/playbooks/nginx.yml

```diff
@@ -0,0 +1,39 @@
+---
+- hosts: server2
+  tasks:
+  
+  - name: "Create web root directory"
+    file: 
+      path: /home/vagrant/html/
+      state: directory
+      owner: vagrant
```

Comment:
vagrant does not look like a good choice for a user. probably leave it out and simple set mode, so that nginx can read it.